tests/cpython-tests/Makefile

```diff
@@ -26,7 +26,7 @@ FS=ext2fs$(VER)
 # Set timeout to 50 mins (to run both the python3.8 test suite and the python3.9 test suite)
 export TIMEOUT=3000
```

Comment:
Please update the comment to include 3.10.

Do we need to increase the time out?

Reply:
The PR pipeline finished successfully for this PR, so I think that might not be necessary? But will add a few minutes just in case